src/rgw/CMakeLists.txt

```diff
@@ -9,6 +9,13 @@ if(WITH_RADOSGW_SELECT_PARQUET)
   message("-- arrow is installed, radosgw/s3select-op is able to process parquet objects")
 endif(WITH_RADOSGW_SELECT_PARQUET)
 
+if(WITH_ARROW_FLIGHT)
+  # next line breaks build
+  # set(ARROW_FLIGHT_LIBRARIES Arrow::ArrowFlight)
```

Comment:
Why does this break the build?

Reply:
in `BuildArrow.cmake`, you named this target `add_library(Arrow::Flight STATIC IMPORTED)`